Makefile

```diff
@@ -288,9 +297,12 @@ prove : $(tests) $(examples)
 prv : ;@$(MAKE) $(MFLAGS) V=$(V) prove
 
 alltests := $(tests) $(examples) $(if $(MFEM_DIR),$(mfemexamples)) $(if $(PETSC_DIR),$(petscexamples))
-prove-all : $(alltests)
+fulltestlist = $(alltests) $(if $(NEK5K_DIR), $(nekexamples))
```

Comment:
Because Nek is built from a bash script rather than in the same way the MFEM and PETSc examples built, I've had trouble making a single Makefile command both provide the commands to build the example and provide the list of files in the `build` directory to run `tap.sh` against. I can investigate further.